example/spid_config/spid_settings.py

```diff
@@ -87,9 +87,11 @@
 
             'endpoints': {
                 'assertion_consumer_service': [
+                    (f'https://previousservice.example.it/acs', SPID_DEFAULT_BINDING),
```

Comment:
Non lo lascerei default nella configurazione, lo commenterei questo.
Quello che serve è una sezione all'interno del README dove guidare i developers in una felice configurazione di ACS aggiuntivi, te lo chiedo in questa revisione prima di unire

Reply:
certamente. In questo momento serve per enfatizzare le modifiche di configurazione